File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/OcsEmr/OCS2015U00GetDiseaseReportInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.OcsEmr
{
    [Serializable]
    public class OCS2015U00GetDiseaseReportInfo
    {
        private String _sangCode;
        private String _sangName;

        public String SangCode
        {
            get { return this._sangCode; }
            set { this._sangCode = value; }
        }

        public String SangName
        {
            get { return this._sangName; }
            set { this._sangName = value; }
        }

        public OCS2015U00GetDiseaseReportInfo() { }

        public OCS2015U00GetDiseaseReportInfo(String sangCode, String sangName)
        {
            this._sangCode = sangCode;
            this._sangName = sangName;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/emr/OCS2015U00GetDiseaseReportInfo.java
package nta.med.data.model.ihis.emr;

public class OCS2015U00GetDiseaseReportInfo {
	private String sangCode;
	private String sangName;
	public OCS2015U00GetDiseaseReportInfo(String sangCode, String sangName) {
		super();
		this.sangCode = sangCode;
		this.sangName = sangName;
	}
	public String getSangCode() {
		return sangCode;
	}
	public void setSangCode(String sangCode) {
		this.sangCode = sangCode;
	}
	public String getSangName() {
		return sangName;
	}
	public void setSangName(String sangName) {
		this.sangName = sangName;
	}
	
}
